### ISSUE TITLE
Question about Trainer inherit.

### ISSUE THREAD

**Issue by Jhin3433** [MARIO-Math-Reasoning/Super_MARIO#31]
Which Trainer in trl should ValueTrainer inherit from? Is it SFTTrainer? I would be very grateful if you could tell me. Thank you.

**Comment by Chen-GX** [comment]
yep, i copy the `SFTTrainer`, modify this trainer as listed in [implementation_details.md](https://github.com/MARIO-Math-Reasoning/Super_MARIO/blob/main/implementation_details.md), and name it as `ValueTrainer`.
